pkgs/crud/crud.go

```diff
@@ -80,6 +82,27 @@ func Retrieve(bctx bahamut.Context, m manipulate.Manipulator, obj elemental.Iden
 	return nil
 }
 
+func newIdentifiable(obj elemental.Identifiable) (elemental.Identifiable, error) {
+
+	objType := reflect.TypeOf(obj)
+
+	for objType.Kind() == reflect.Ptr ||
```

Comment:
we only allow a pointer to be passed in Update. I don't think we need to check for anything else.